src/main/resources/application.yaml

```diff
@@ -198,6 +198,16 @@ govnotify:
       other:
         email: 9a93d07a-8f0e-4057-9b5d-3f567f2c5fb9
 
+    applicationGrantedApplicant: 565437ef-ab10-4c1a-bf1f-de7306662906
```

Comment:
it seems that changes do not keep above layout for template ids

Reply:
Will be done in 2974